src/cards/suffocate.ts

```diff
@@ -24,13 +25,19 @@ function add(unit: Unit.IUnit, underworld: Underworld, prediction: boolean, quan
       }
     }
   });
-  const lastTurnsLeftToLive = modifier.turnsLeftToLive;
-  modifier.turnsLeftToLive = 1 + Math.ceil(unit.health / 20 / modifier.quantity)
-  // Ensure that casting Suffocate doesn't ever increase the turns left
-  if (lastTurnsLeftToLive) {
-    modifier.turnsLeftToLive = Math.min(modifier.turnsLeftToLive, lastTurnsLeftToLive);
+
+  modifier.buildup = Math.floor(10 * Math.pow(2, (modifier.quantity - 1) / 2));
+
+  if (modifier.buildup >= unit.health) {
```

Comment:
This is very interesting because it means suffocate is more powerful if you hurt them and less powerful if you heal them.  I think I like it.
TODO: I'll need to make localization changes to any of these spells when their functionality changes.

Reply:
We could always change it to use max hp too, if you don't want it to work like a 2nd poison. I think it's nice either way.